projects/angular/src/accordion/_accordion.clarity.scss

```
@@ -97,7 +98,7 @@
   }
 
   .clr-accordion-status {
-    width: tokens.$cds-global-space-11;
+    width: density.$clr-base-layout-space-xl;
     display: inline-block;
```

The caret is not centered.

```suggestion
    display: inline-flex;
```